src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+                System.out.println("____________________________________________________________\n"+
+                                    "Bye. Hope to see you again soon!\n"+
+                                    "____________________________________________________________");
+                break;
+            }else if (command.equals("list")) {
+                System.out.println("____________________________________________________________");
+                listTasks();
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("done")) {
+                doneTasks(command);
+                System.out.println("____________________________________________________________");
+            } else if (command.contains("todo")) {
+                System.out.println("____________________________________________________________");
+                todoTask(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("deadline")) {
+                System.out.println("____________________________________________________________");
+                deadlineTasks(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("event")) {
+                System.out.println("____________________________________________________________");
+                eventTask(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("delete")) {
+                System.out.println("____________________________________________________________");
+                deleteTask(command);
+                System.out.println("____________________________________________________________");
+
+            } else {
+                System.out.println("____________________________________________________________");
+                System.out.println("OOPS!!! I'm sorry, but I don't know what that means :-(");
+                System.out.println("____________________________________________________________");
+            }
+        }
+
+
+    }
+
+
+    private static void createDirAndFile() {
+        try {
+            System.out.println("Creating file: data/duke.txt");
+            File newDir = new File("data");
+            newDir.mkdir();
+            File f = new File("data/duke.txt");
+            if (f.createNewFile()) {
+                System.out.println("File created: " + f.getName());
+            } else {
+                System.out.println("File already exists.");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+    }
+
+        private static void deadlineTasks(String command) {
+
+        try {
```

Comment:
You could consider the indentation conventions [here](https://se-education.org/guides/conventions/java/basic.html#layout).